addons/website_cf_turnstile/static/src/js/turnstile.js

```diff
@@ -29,8 +31,9 @@ publicWidget.registry.s_website_form.include({
                             throw error;
                         }
                     </script>
-                    <script class="s_turnstile" src="https://challenges.cloudflare.com/turnstile/v0/api.js"></script>
-                `).insertAfter('.s_website_form_send, .o_website_form_send');
+                `);
+                $(turnstileContainer).add(turnstileErrorGlobalScript).add($(turnstileScript))
```

Comment:
`turnstileContainer` is, I think, already an element so you could add them like you would in regular JS.

Reply:
I tried to keep it logically close to the original in case some jquery sideeffect had a visible impact. But indeed we can convert to standard js dom api and see if framework has an objection later.